mods/farming/nodes.lua

```diff
@@ -123,7 +129,15 @@ minetest.register_node("farming:desert_sand_soil", {
 	description = S("Desert Sand Soil"),
 	drop = "default:desert_sand",
 	tiles = {"farming_desert_sand_soil.png", "default_desert_sand.png"},
-	groups = {crumbly=3, not_in_creative_inventory = 1, falling_node=1, sand=1, soil = 2, desert = 1, field = 1},
+	groups = {
+		crumbly = 3,
+		not_in_creative_inventory = 1,
+		falling_node = 1,
+		sand = 1,
+		soil = 2,
+		desert = 1,
+		field = 1
```

Comment:
> No other mods use similar group formatting. Are you sure this is better?

You forgot these lines.

Reply:
Thanks. I've also changed groups code in api.lua.